### PR TITLE
Updating Build/Deploy automations

### PR DESCRIPTION
**Pull request by sargentfrancesca**

### CODE REVIEW

**Comment by andylolz** [comment]
@sargentfrancesca I’ve no idea if this is of any use, but I thought I’d just flag this comment / demo for building and deploying the reference site using Travis CI:
https://github.com/IATI/IATI-Standard-SSOT/issues/297#issuecomment-473092152

It probably won’t work with the latest python3 changes, but I guess it wouldn’t take much to fix.

File: deployment/build_iati_standard.sh [hunk 1]
```diff
@@ -51,14 +62,42 @@ echo -e "UPDATED DOCS & TEMPLATES \n\n"
...
     echo -e "DONE CODE PULLS: $f \n\n"
 
     # Set-up the environment and repository dependencies
+    # Remove existing pyenv to replace with new
+    if [ -d "pyenv/" ]; then
+        echo "REMOVING EXISTING PYENV" 
+        rm -Rf pyenv/
+    fi
+
+    if [ "$PYTHON_2" = true ]; then
```
Comment (samuele-mattiuzzo) [inline deployment/build_iati_standard.sh]: this is good, do we want to keep it though? should we not be a bit more annoying and just force the users to have py3?

**Comment by samuele-mattiuzzo** [comment]
This is really good @sargentfrancesca 👍

File: deployment/build_iati_standard.sh (hunk 1 above)
Comment (sargentfrancesca) [reply deployment/build_iati_standard.sh]: this flag can be removed when py3 upgrades have been merged and deployed, otherwise not needed!

**Comment by sargentfrancesca** [comment]
@andylolz ah, thank you so much! This is great, and really useful. 

We're still scoping how our continuous delivery might look for the SSOT - the amendments here are as close as I could get to consolidating what exists in a more robust way whilst not abstracting too far from the purpose of this set of updates. We're actually resuming work on containerising the [IATI Standard Website](https://github.com/IATI/IATI-Standard-Website) (https://github.com/IATI/IATI-Standard-Website/compare/master...add-docker) at the moment as a starting point for consolidating build and deploy processes throughout our tools. I'm sandpitting Dokku as a PaaS (https://github.com/IATI/IATI-Standard-Website/compare/add-docker...add-dokku). I can see your snippet being really useful when we finally put this into practise.

**Pull request merged**